src/crawlee/base_storage_client/_models.py

```diff
@@ -230,3 +231,29 @@ class BatchRequestsOperationResponse(BaseModel):
 
     processed_requests: Annotated[list[ProcessedRequest], Field(alias='processedRequests')]
     unprocessed_requests: Annotated[list[UnprocessedRequest], Field(alias='unprocessedRequests')]
+
+
+@docs_group('Data structures')
+class InternalRequest(BaseModel):
+    """Represents an request in queue client."""
+
+    model_config = ConfigDict(populate_by_name=True)
+
+    id: str
+
+    unique_key: str
+
+    order_no: Decimal | None = None
+
+    handled_at: datetime | None
+
+    request: Annotated[Request, Field(alias='json_')]
```

Comment:
Could you add docstrings explaining the fields? Mostly what are `order_no` & `request` for. Also mention the purpose of the `json_` alias.